vuln/npm/518.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": 518,
+  "title": "Path Traversal",
+  "overview": "[Total.js] Path traversal vulnerability allows to read files outside public directory",
+  "created_at": "2019-11-30",
+  "updated_at": "2020-04-25",
+  "publish_date": "2020-04-25",
+  "author": {
+    "name": "Visat",
+    "website": null,
+    "username": "visat"
+  },
+  "module_name": "Other module",
```

Comment:
Shouldn't this be the name of the package?
```suggestion
  "module_name": "total.js",
```

Reply:
Indeed, thanks!